queries/models/bilhetagem/schema.yml

```
@@ -481,4 +481,4 @@ models:
       - name: id_execucao_dbt
         description: "{{ doc('id_execucao_dbt') }}"
         data_type: string
-        quote: true
+        quote: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)